project/dimensions/subsectors.csv

```diff
@@ -1,20 +1,56 @@
 id,name
 unspecified,Unspecified
-bev_compact,BEV Compact
-bev_midsize,BEV Midsize
-bev_pickup,BEV Pickup
-bev_suv,BEV SUV
-phev_compact,PHEV Compact
-phev_midsize,PHEV Midsize
-phev_pickup,PHEV Pickup
-phev_suv,PHEV SUV
-bev_light_medium,BEV Light-Medium
-bev_medium,BEV Medium
-bev_non_freight_truck,BEV Non-freight truck
-bev_heavy,BEV Heavy
-bev_school_bus,BEV School Bus
-bev_transit_bus,BEV Transit Bus
-light_rail,Light Rail
+LDV+Personal_LDV+Compact+BEV_200, LDV+Compact+BEV_200
```

Comment:
This may not be what @elainethale had in mind.

Reply:
I see - I'll revert this and create the dataset's dimensions and a mapping.